source/_integrations/evohome.markdown

```diff
@@ -34,10 +34,12 @@ To set up this integration, add the following to your `configuration.yaml` file:
 ```yaml
 # Example configuration.yaml entry
 evohome:
-  username: "YOUR_USERNAME"
-  password: "YOUR_PASSWORD"
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
```

Comment:
These changes are not correct. String should be enquotes, as per our YAML documentation standards.

../Frenck

Reply:
Which is preferable - documentation that meets your standards, or documentation that causes the software to actually work?

Reply:
If it doesn't work, then it sounds like a bug. As there is no difference in those syntaxes for the outcome. Both are considered a string by YAML. As a matter of fact, enquoting is better, as it is more explicit (hence it is the standard for documenting string values).

../Frenck

Reply:
I've retested and added the quotes back in, and it still works so this PR can be ditched. For whatever reason HA didn't pick up or start working on the first restart, but on the second restart (when I had removed the quotes) it did  - my apologies.